in-progress/7520-testnet-overview.md

```diff
@@ -0,0 +1,391 @@
+
+Only the owner of the Incentives contract can mint TST.
+
+It has a rate limiter on minting.
+
+The Incentives contract will be immutable.
+
+### Governance Contract
+
+The Governance Contract owns the Incentives contract.
+
+TST holders can lock their TST in the Governance Contract to vote on proposals.
+
+Proposals can only be submitted by the PendingProposals Contract.
+
+Proposals must garner X% of the total locked TST to be ratified.
+
+There will be a time delay between ratification and execution.
+
+### Registry Contract
+
+The Registry Contract will be able to keep track of the current/canonical and historical Instances.
+
+An Instance will be comprised of:
+- A Rollup Contract, which is the main contract that handles the rollup of transactions.
+- A Data Availability Oracle, which is responsible for answering if the preimage of commitments have been made available.
+- An Inbox, responsible for receiving messages from L1 and making them available L2.
+- An Outbox, responsible for receiving messages from L2 and making them available on L1.
+
+### Rollup Contract
+
+The initial Rollup Contract will require holders of TST to stake their tokens to become validators.
+
+The initial Rollup Contract will maintain a balance of TST to be used for rewards.
+
+### PendingProposals Contract
+
+The Proposals Contract keeps track of governance proposals and votes.
+
+It watches for proposal signals in the Registry's canonical Instance.
+
+When M of the previous N blocks contain the same proposal, it is submitted to Governance.
+
+## Forced Inclusions
+
+Deployments will have a mechanism for forced inclusions of transactions in the canonical chain.
+
+### Open Questions
+
+- How does it work?
+- Can this mechanism be tied to the based sequencing mechanism?
+
+## Aztec Node
+
+Aztec Labs will provide a reference implementation of the Aztec Node, which will be used to run the Aztec Network.
+
+It will have 3 primary modes of operation:
+
+- Proposer/Validator: responsible for proposing new blocks and validating them
+- Prover: responsible for orchestrating the creation of various proofs
+- Full Node: follows along, responsible for propagating transactions and blocks
+
+The Aztec Node will have a web interface for monitoring key metrics.
+
+## Chains, slots, and epochs
+
+There will be three chains in the Aztec Network:
+
+- The Pending Chain
+- The Proven Chain
+- The Finalized Chain
+
+All three chains will be effectively managed by the Aztec Node and the L1 contracts, and will have different guarantees.
+
+Time will be divided into slots, which will be grouped into epochs.
+
+Each slot will have a proposer, who is responsible for proposing a block of transactions.
+
+Each epoch will have a set of validators, who add economic security to the Pending Chain by providing signatures on proposed blocks; ultimately the Pending Chain is a UX feature that allows users to see their transactions with _some guarantee_ before they are proven.
```

Comment:
Can we break out what is being signed on here. i.e The validator is signing over the fact that they have the proof for each transaction, and the proof passes.

We can then add a later training wheel module.

Reply:
(I should have kept reading)